bindings/java/iota-wallet-java/lib/build.gradle

```diff
@@ -58,11 +72,15 @@ jar {
         include '*.so', '*.dylib', '*.dll'
     }
 
+    from('native/target/' + findProperty('buildTarget') + '/release') {
```

Comment:
Not release anymore.
Also, change all mentions of release in this file (when release means profile ofc, not actual release binary)